src/cljs/main/broadfcui/page/workspace/summary/tab.cljs

```diff
@@ -283,10 +287,12 @@
 (react/defc Summary
   {:get-initial-state
    (fn []
-     {:sidebar-visible? true})
+     {:sidebar-visible? true
+      :label-id (gensym "status")
+      :body-id (gensym "summary")})
```

Comment:
I would normally put these into `locals` since I wouldn't ever expect them to change, but I do like the simplicity of this and maybe that is not an important distinction.